rust/src/detect/flow.rs

```diff
@@ -48,11 +48,13 @@ pub struct DetectFlowBytes {
     pub dir: DetectFlowDir,
 }
 
+
 fn detect_parse_flow_direction(i: &str) -> IResult<&str, DetectFlowDir> {
     let (i, fd) = alt((
         value(DetectFlowDir::DETECT_FLOW_TOSERVER, tag("toserver")),
         value(DetectFlowDir::DETECT_FLOW_TOCLIENT, tag("toclient")),
         value(DetectFlowDir::DETECT_FLOW_TOEITHER, tag("either")),
+        value(DetectFlowDir::DETECT_FLOW_TOBOTH, tag("both")),
```

Comment:
Is this acceptable, @catenacyber ? It can be done by specifying both `toserver` and `toclient` w a keyword indeed, but just one keyword instance like this. I don't have any strong opinions about this. Will adjust as per your guidance.